### DISCUSSION TITLE
Milvus v1.1 support cluster mode

### DISCUSSION THREAD

**Discussion by xiaojuan8** [milvus-io/milvus#15641] [Q&A and General discussion]
Hi，is any version of milvus support both distributed clusters and GPU mode？No info found on official doc：https://milvus.io/docs
thanks.

**Comment by yhmo** [comment]
Milvus 1.x has a middleware(written by python) to build a cluster mode service: https://github.com/milvus-io/milvus/tree/1.1/shards

**Reply by xiaojuan8** [reply]
👌，thanks.

**Comment by yhmo** [comment]
Just remind, the 1.x distributed mode has some limitations:
1. all nodes must share the same storage and meta service
2. it has a bug to reduce read nodes, for example, start with 4 read nodes. Search could fail after shutdown 2 read nodes.
3. only allow one write node in the cluster

**Reply by xiaojuan8** [reply]
OK, I'll try it.
